src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -67,68 +86,68 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     setPopover(false);
   };
 
-  const workspaceToItem = (workspace: WorkspaceObject) => {
+  const workspaceUseCaseToItem = (
+    workspace: WorkspaceObject,
+    index: number,
+    workspaceURL: string
+  ) => {
+    const useCase = workspace.features
+      ?.map(getUseCaseFromFeatureConfig)
+      .filter(isNotNull)
+      .map((uc) => {
+        return {
+          name: uc,
+          // TODO: this should be a link to the use case overview page.
+          onClick: () => {
+            window.location.assign(workspaceURL);
```

Comment:
question: Is this the right way to do client side navigation?

Reply:
Yes, we can use this way in `contextmenu`.